### DISCUSSION TITLE
Improve the function brackets completion

### DISCUSSION THREAD

**Discussion by Vincent0706** [microsoft/pylance-release#3609] [Enhancement]
I hope the pylance can distinguish whether the function need add brackets. Sometimes, the brackets have been input and we just need input the funtion name. But when you open the function brackets completion, pylance will add the brackets everytime. 

https://user-images.githubusercontent.com/82581024/139382092-2c234562-7e68-483b-a98a-566b53c379f5.mp4

**Comment by ldorigo** [comment]
I agree that this is pretty irritating and happens all the time (and most other language servers/editors have a built-in check that only adds the parentheses if they are not already present).

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
